ui/raidboss/data/03-hw/dungeon/sohm_al_hard.js

```diff
@@ -0,0 +1,314 @@
+'use strict';
+
+[{
+  zoneRegex: {
+    en: /^Sohm Al \(Hard\)$/,
+  },
+  zoneId: ZoneId.SohmAlHard,
+  timelineFile: 'sohm_al_hard.txt',
+  timelineTriggers: [
+    {
+      id: 'Sohm Al Hard Wild Horn',
+      regex: /Wild Horn/,
+      beforeSeconds: 4,
+      condition: function(data) {
+        return data.role == 'tank' || data.role == 'healer';
+      },
+      response: Responses.tankBuster(),
+    },
+  ],
+  triggers: [
+    {
+      // The actual damage is 1C31, but the windup for the damage
+      // occurs between 1C30 and 1C31.
+      id: 'Sohm Al Hard Inflammable Fumes',
+      netRegex: NetRegexes.ability({ id: '1C30', source: 'The Leightonward', capture: false }),
+      netRegexCn: NetRegexes.ability({ id: '1C30', source: '莱顿瓦德', capture: false }),
+      netRegexDe: NetRegexes.ability({ id: '1C30', source: 'Hortigolem', capture: false }),
+      netRegexFr: NetRegexes.ability({ id: '1C30', source: 'Chortocyon', capture: false }),
+      netRegexJa: NetRegexes.ability({ id: '1C30', source: 'レイトンワード', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      // Both the small and large Spore Sacs use Glorious Blaze.
+      // However, it's not the same ability.
+      id: 'Sohm Al Hard Glorious Blaze',
+      netRegex: NetRegexes.startsUsing({ id: '1C32', source: 'Spore Sac', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '1C32', source: '孢囊', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '1C32', source: 'Sporensack', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '1C32', source: 'Sac de spores', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '1C32', source: 'スポアサック', capture: false }),
+      suppressSeconds: 5,
+      infoText: {
+        en: 'Away from large pod',
+      },
+    },
+    {
+      // The actual effect being checked here is Heavy.
+      id: 'Sohm Al Hard Excretion',
+      netRegex: NetRegexes.gainsEffect({ effectId: '0e' }),
```

Comment:
For consistency with other ids, can you make this `0E`.  Names might be capitalized inconsistently, but ids are capital everywhere else.